docs/adr/0002-testing-strategy.rst

```diff
@@ -1,4 +1,4 @@
-[0002] Better testing framework for entire Raiden suite (client, MS and PFS)
+ Better testing framework for the entire Raiden suite (client, MS and PFS)
```

Comment:
Unnecessary space in the beginning of the line.

Reply:
Will fix and then merge. Thanks @palango